platform-update-agent/internal/installer/installer.go

```diff
@@ -94,10 +94,6 @@ func (i *Installer) ProvisionInbm(ctx context.Context) error {
 		return fmt.Errorf("failed to execute shell command - %v", err)
 	}
 
-	if err := i.modifyConfiguration(ctx); err != nil {
```

Comment:
what is it doing this change? Is the PR description properly updated ?

Reply:
There is a timeout issue observed in this operation, which is delaying the overall PUA bring-up time. The purpose of this operation is to modify the INBM configuration file to remove certain software checks during system update. We decide to remove this operation from the PUA and let the OM to handle it. The PR for OM has been created at https://github.com/open-edge-platform/infra-onboarding/pull/47

Reply:
PR description updated.